### PR TITLE
Issue #2563791 by dakala: Apply coding standards, fix coding style, p…

### PR DESCRIPTION
**Pull request by mglaman**
…hpDoc, and comments

WIP!

### CODE REVIEW

**Comment by mglaman** [comment]
mglaman/profile@3a1c4c1 stopped at FILE: ...home/travis/build/bojanz/profile/src/Entity/ProfileInterface.php

File: src/Controller/ProfileController.php [hunk 1]
```diff
@@ -71,6 +72,19 @@ public function addPageTitle(ProfileTypeInterface $profile_type) {
     return $this->t('Create @label', ['@label' => $profile_type->label()]);
   }
 
+  /**
+   * Provides profile create form.
+   *
+   * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+   *    The route match.
```
Comment (bojanz) [inline src/Controller/ProfileController.php]: These param descriptions are indented one space too many :)

File: src/Access/ProfileAccessCheck.php [hunk 2]
```diff
@@ -20,10 +20,10 @@
 class ProfileAccessCheck implements AccessInterface {
 
   /**
- * The entity manager.
- *
- * @var \Drupal\Core\Entity\EntityManagerInterface
- */
+   * The entity manager.
+   *
+   * @var \Drupal\Core\Entity\EntityManagerInterface
```
Comment (bojanz) [inline src/Access/ProfileAccessCheck.php]: We should be replacing the entity manager with the entity type manager wherever we can.

Comment (mglaman) [inline src/Access/ProfileAccessCheck.php]: Should we move that to its own ticket, since it involves modifying the services.yml, too?

Comment (bojanz) [inline src/Access/ProfileAccessCheck.php]: Sure, makes sense.

Comment (mglaman) [inline src/Access/ProfileAccessCheck.php]: Made follow up issue: https://www.drupal.org/node/2617974

**Comment by mglaman** [comment]
@bojanz this should be all set now. well according to phpcs, minus some minor things.

**Pull request merged**